react-twitter-save-data-loading/README.md

```diff
@@ -1,3 +1,20 @@
-## Glitch Source
-* [Link to Glitch App](https://anton-karlovskiy-react-save-data-twitter.glitch.me/)
-* [Link to Project on Glitch](https://glitch.com/~anton-karlovskiy-react-save-data-twitter/)
\ No newline at end of file
+
+# Demo: React-twitter save-data resource loading in React
+
+[React Tweet](https://github.com/mannynotfound/react-tweet) is an demo application looking like Twitter website.
+
+This version of React twitter uses Save-Data Client Hint to determine whether to load low-fidelity or high-fidelity images based on the user's Save-Data option on Android Chrome (e.g Lite Mode On -> lo-fi, Lite Mode Off -> hi-fi).
+
+[Live Demo](https://adaptive-loading.web.app/react-twitter-save-data-loading/)
```

Comment:
I just tested https://adaptive-loading.web.app/react-twitter-save-data-loading/ using Lite Mode on Android and it's working really great!